src/libs/shared/utils/is-defined.util.ts

```diff
@@ -1,3 +1,6 @@
 // @TODO: 15) napisz isDefined type guard który będzie sprawdzał czy obiekt nie ma wartości undefined lub null i odpowiednio go typował
 
-export const isDefined = () => true;
+export const isDefined = (object: any): boolean => {
```

Comment:
warunek jest spoko, tyle że to nie jest type guard, zamiast zwracanego typu "boolean", powinno być "object is boolean"